modules/aruco/samples/calibrate_camera_charuco.cpp

```diff
@@ -57,8 +57,8 @@ const char* about =
 const char* keys  =
         "{w        |       | Number of squares in X direction }"
         "{h        |       | Number of squares in Y direction }"
-        "{sl       |       | Square side lenght (in pixels) }"
-        "{ml       |       | Marker side lenght (in pixels) }"
+        "{sl       |       | Square side length (in meters) }"
+        "{ml       |       | Marker side length (in meters) }"
```

Comment:
But this is the code that generates the calibration parameters. To generate these for a camera system you need the units in the real world context only then will we get a relation between pixels and real world units like millimetres. 

The input parameters too down in the parser code, accept **float** instead of **int**.